.github/scripts/ci-checkout.sh

```diff
@@ -0,0 +1,11 @@
+# ci-checkout.sh
+set -ex
+
+. $(dirname "$0")/common.sh
+
+OPENJDK_URL=`sed -n 's/^openjdk_repo.=."\(.*\)"$/\1/p' $BINDING_PATH/mmtk/Cargo.toml`
+OPENJDK_VERSION=`sed -n 's/^openjdk_version.=."\(.*\)"$/\1/p' $BINDING_PATH/mmtk/Cargo.toml`
```

Comment:
Regular expression is not stable when paring this highly-structured TOML file.  The follwing one-liner can be useful:
```bash
cargo read-manifest | python -c 'import json, sys; print(json.load(sys.stdin)["metadata"]["openjdk"]["openjdk_repo"])'
```